src/app/components/InventoryItem/index.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  Checkbox,
+  IconButton,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  TextField,
+} from '@mui/material';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  deleteInventoryItem,
+  updateInventoryItem,
+} from '../../reducers/inventoryReducer';
+import './index.css';
+import DeleteIcon from '@mui/icons-material/Delete';
+import DoneIcon from '@mui/icons-material/Done';
+
+type InventoryItemType = {
+  id: number;
+  title: string;
+  checked: boolean;
+};
+
+type Props = {
+  item: InventoryItemType;
+};
+
+export default (props: Props): JSX.Element => {
+  const dispatch = useDispatch();
+
+  const [editing, setEditing] = useState<boolean>(false);
+
+  const handleClick = () => {
+    setEditing((current) => !current);
+  };
+
+  const handleCheck = (
+    _: React.ChangeEvent<HTMLInputElement>,
+    checked: boolean
+  ) => {
+    dispatch(
+      updateInventoryItem({
+        ...props.item,
+        checked,
+      })
+    );
+  };
+
+  const handleUpdate = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const title = event.target.value;
+    dispatch(
+      updateInventoryItem({
+        ...props.item,
+        title,
+      })
+    );
+  };
+
+  const handleDelete = () => {
+    dispatch(
+      deleteInventoryItem({
+        ...props.item,
+      })
+    );
+  };
+  return (
+    <ListItem>
+      <ListItemIcon>
+        <Checkbox
+          edge="start"
```

Comment:
naming `edge` is not clear here. What does it do?

Reply:
This isn't something I wrote, it is a MateriaI UI property.